scheme/arm-cca/realm.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package arm_cca
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/veraison/ccatoken"
+	"github.com/veraison/ear"
+	"github.com/veraison/services/handler"
+	"github.com/veraison/services/log"
+	"github.com/veraison/services/scheme/common/arm"
+	realm "github.com/veraison/services/scheme/common/cca/realm"
```

Comment:
is the alias needed?

Reply:
see https://github.com/veraison/services/pull/233/files#r1645622827

Reply:
Fixed in https://github.com/veraison/services/commit/6a4041ff7758e0830e5215ba5e7ad258fa4464ea